pac4j-oauth/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
```

Comment:
Outputting DEBUG logs for tests seems to be the right thing to do. Though, I have limit the logs produced by tests in order to have smaller logs file size in Travis. See: https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/test/resources/logback.xml

In short, can you remove this file? Change the one in the core when you need more logs...

Reply:
Will do.